<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deposit Calc Result</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #3d3d3d;</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="2">
     <widget class="QLabel" name="label_sum_hide_income">
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: lightgray;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Amount of withdrawal ........</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_17">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background: lightgray;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_20">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background: #fe6f5e;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_hide_income">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Income after taxes ..........</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_hide_taxed_sum">
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_sum_deposit">
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: orange;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: #fe6f5e;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Withheld taxes ...............</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: orange;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Deposit amount ...............</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_18">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background: #007FFF;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="label_final_sum">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;
background: rgba(175, 48, 173, 110);
font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>final result</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: bold 20px;
color: green;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Amount of Replenishment ....</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_sum_withdrawal">
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color:  lightgray;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: #007FFF;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Interest charges .............</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_sum_replenishment">
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: green;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_hide_taxed">
      <property name="minimumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>28</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Taxed .........................</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="Line" name="line">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="label_sum_withheld_taxes">
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color:  #fe6f5e;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_sum_interest_charges">
      <property name="styleSheet">
       <string notr="true">font: bold 20px;
color: #007FFF;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_16">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background: green;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="minimumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>10</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background: orange;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_final_sum_2">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>290</width>
        <height>30</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
border-radius: 10px;
background: rgba(175, 48, 173, 110);
font: bold 20px;
color: white;
font-family: &quot;Comic Sans MS&quot;;</string>
      </property>
      <property name="text">
       <string>Deposit amount with interest:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
